Etna Upgrade Motivation (#54)
ava-labs/avalanche-academy: content/guide/avalanche9000-motivation.mdx

---
title: What is the Motivation behind the Avalanche9000 Upgrade?
description: Learn about the largest network upgrade of Avalanche, that will lower the barrier to launch a blockchain and enable permissionless vlaidation of L1s.
date: 2024-08-15
authors: [martin_eckardt, owenwahlgren]
topics: [Avalanche]
---

Avalanche9000 will be the largest network upgrade of Avalanche to date. This upgrade will introduce a new concept called L1s, which will replace the current Subnet model. In this article we will discuss the current limitations of Subnets, and how the changes suggested in [ACP-77](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/77-reinventing-subnets/README.md) will address these limitations in the Etna Upgrade.

<YouTube id="eczkjrGG_As" />
^^ placeholder

## ACP-77

### How Subnets Work Today

![Subnets](/common-images/multi-chain-architecture/acp-77/subnets.png)

The Avalanche network has a large set of validators who all participate in running and validating the Primary Network.

The Primary Network consists of the C-Chain, P-Chain, and X-Chain, and is where most of the transactions on Avalanche are processed.

> In order to scale horizontally and offer more blockspace, a subset of validators can opt-in to validate an additional blockchain in parallel with the Primary Network, we call this a Subnet.

Reviewer: ```suggestion
In order to scale horizontally and offer more blockspace, a subset of validators can opt-in to validate an additional blockchain in parallel with the Primary Network: we call this a Subnet.
```

### Subnets -> L1s

> With ACP-77 Subnets will become their own Layer 1 blockchain.

Reviewer: ```suggestion

With ACP-77, Subnets will become their own Layer 1 blockchain.
```

![L1s](/common-images/multi-chain-architecture/acp-77/l1s.png)

> Instead of using subsets of validators from the Primary Network - we now use a new distinct set of validators.

Reviewer: ```suggestion
Instead of using a subset of Primary Network Validators, new L1s can now use a distinct set of validators.
```
Avoid the use of "we" "us" "you" and other pronouns.

This change maintains the short time of finality and high throughput Avalanche Subnets are known for while also enabling a new set of features with options for completely customizable staking options, gas tokens, and governance.

### Current Economic Barriers

> ![Staking Requirements](/common-images/multi-chain-architecture/acp-77/subnets-staking-requirements.png)
> The Primary Network currently has a staking requirement of 2,000 AVAX.

Reviewer: ```suggestion
![Staking Requirements](/common-images/multi-chain-architecture/acp-77/subnets-staking-requirements.png)

The Primary Network currently has a staking requirement of 2,000 AVAX.
```

If we assume a price of $20 per AVAX, this would be a $40,000 staking requirement per validator.

> To launch a Subnet today, validators must meet the staking requirement for each subsequent Subnet they choose to validate. This is typically a native token to the Subnet.
> This model can quickly lead to a high economic barrier of entry in creating new Subnets as validators must meet the 2,000 AVAX requirement in addition to the staking requirements of any Subnets they want to validate.

Reviewer: ```suggestion
This model has created a significant economic barrier for the creation of new Subnets as validators must meet the 2,000 AVAX requirement, and a majority of Subnets prefer to run with a minimum of 5 validators.
```

No subnets in production or on fuji have ever required validators to stake, as many remained permissioned from the get go.

### Lowering Economic Barriers
![L1 Staking Requirements](/common-images/multi-chain-architecture/acp-77/l1s-staking-requirement.png)

> This upgrade will remove the requirement to stake 2,000 AVAX on the Primary Network, and instead only require validators to stake on L1 with their native token.
> This will lower the economic barrier to entry, and enable more validators to participate in the network.

Reviewer: ```suggestion
This upgrade will remove the requirement for L1 validators to stake 2,000 AVAX on the Primary Network, and instead only require validators to abide by the staking requirements defined by the L1 they wish to validate.
This will remove the significant economic barrier, giving more sovereignty to L1 creators and validators.
```

### Current Running Costs
![Subnet Running Cost](/common-images/multi-chain-architecture/acp-77/subnet-running-cost.png)

Currently, validators must validate the Primary Network in addition to each Subnet they are validating. This can lead to higher running costs for validators as they must maintain machines that can handle the load of both networks.

### Lower Running Costs
![L1 Running Cost](/common-images/multi-chain-architecture/acp-77/l1-running-cost.png)

> With ACP-77, validators will only need to validate the L1. This will lead to lower running costs for validators as they will not need to maintain machines that can handle the load of both the Subnets and Primary Network.

Reviewer: ```suggestion

With ACP-77, validators will only need to validate the L1. This will lead to lower running costs for validators as they will not need to maintain machines that can handle the load of both the Subnets and the Primary Network.
```

### Current Compliance Barriers
![Subnet Compliance](/common-images/multi-chain-architecture/acp-77/subnets-compliance.png)

> Many Institutions that want to adopt blockchain are struggling with exposure to public permissionless blockchains.
> The Primary Network is a public permissionless blockchain, and as such, it is not suitable to build on for many institutions.

Reviewer: ```suggestion
Many institutions that want to adopt blockchain are struggling with exposure to public permissionless blockchains.
The Primary Network is a public permissionless blockchain, and as such, it is not suitable to build on for many institutions.
```

this statement doesn't explain _why_ institutions are hesitant or prohibited from building on permissionless chains. Let's elaborate

> While institutions can launch their own Subnet as a private, proof of authority blockchain, they are still exposed to the Primary Network in some manner.
> 
> This exposure can be circumvented by performing a partial sync of the Primary Network, but is an incomplete solution.
> ### Better Institutional Compliance
> ![L1 Compliance](/common-images/multi-chain-architecture/acp-77/l1s-compliance.png)

Reviewer: ```suggestion
While institutions can launch their own Subnet as a private, proof of authority blockchain, they are still exposed to the Primary Network in some manner.

This exposure can be mitigated by performing a partial sync of the Primary Network, but is an incomplete solution.

### Better Institutional Compliance

![L1 Compliance](/common-images/multi-chain-architecture/acp-77/l1s-compliance.png)
```
need to explain why exposure to the primary network doesn't favor institutions

> Since validator sets of L1s are now distinct and seperate from the Primary Network, institutions can now launch their own L1s (be it public or private) without exposure to the public permissionless Primary Network.

Reviewer: ```suggestion
Since the validator sets of L1s will be distinct and separate from the Primary Network, institutions will be able to launch independent L1s (be it public or private) that are not required to participate in the Primary Network.
```

> ### Current Fault Isolation
> ![Fault Isolation](/common-images/multi-chain-architecture/acp-77/subnets-primary-network-congestion.png)

Reviewer: ```suggestion
### Current Fault Isolation

![Fault Isolation](/common-images/multi-chain-architecture/acp-77/subnets-primary-network-congestion.png)
```

Currently, if there is high demand for the Primary Network, it can lead to congestion and higher gas fees. 
Because Subnet validators must also validate the Primary Network, their machines are exposed to the same congestion which can have a consequential effects on Subnet performance.

> ### New Fault Isolation
> ![L1s Fault Isolation](/common-images/multi-chain-architecture/acp-77/l1s-primary-network-faults.png)

Reviewer: ```suggestion
### New Fault Isolation

![L1s Fault Isolation](/common-images/multi-chain-architecture/acp-77/l1s-primary-network-faults.png)
```

Because validator sets are now distinct, congestion on the Primary Network will not affect the performance of L1s and vice versa.

### Conclusion

ACP-77 is the largest network upgrade of Avalanche to date. This change will enable more developers to get inexpensive block space for their applications while also enabling institutions to launch their own L1s with better compliance and fault isolation.

> You can learn more in depth about ACP-77 by reading the [ACP-77 Proposal](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/77-reinventing-subnets/README.md)

Reviewer: ```suggestion
You can learn more in depth about ACP-77 by reading the [ACP-77 Proposal](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/77-reinventing-subnets/README.md).
```
